WDAC-Policy-Wizard/app/Properties/AssemblyInfo.cs

```diff
@@ -36,5 +36,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.6.6.0")]
```

Comment:
is there a way to isolate this constant to a separate variable that you only have to change once?

Reply:
this still is relevant